### ISSUE TITLE
Boot Freezes on A Black Screen in VMware Workstation.

### ISSUE THREAD

**Issue by daemonspudguy** [SerenityOS/serenity#3893]
I don't know what it's doing.
![image](https://user-images.githubusercontent.com/24438557/97791472-3191e500-1ba9-11eb-8c52-572d0d4b30bc.png)

**Comment by Lubrsi** [comment]
IIRC from when I tried it, there's two issues:
- The scheduler gets stuck in the idle loop
- We need to write a new video driver for VMWare SVGA 2

**Comment by tomuta** [comment]
>     * The scheduler gets stuck in the idle loop

This sounds like exactly what happens when trying to boot it in Bochs. At least in Bochs it seems like the HPET timer never fires.

**Comment by daemonspudguy** [comment]
Still completely borked.

**Comment by tomuta** [comment]
> Still completely borked.

Yeah because of other bugs. PS/2 has major problems (I'm working on that), and then sounds like we don't have an appropriate svga driver for vmware. You're welcome to contribute ;-)

**Comment by Lubrsi** [comment]
I noticed it does try to use MBVGA on VMware, not sure if that'll be enough before making an SVGA driver.

**Comment by Lubrsi** [comment]
Plus there's also an IDE bug that needs to be found and fixed.

**Comment by linusg** [comment]
Probably related to #3747
